pkgs/by-name/qd/qdiskinfo/package.nix

```
@@ -9,13 +9,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "QDiskInfo";
```

I think
```suggestion
  pname = "qdiskinfo";
```
is more canonical.

---

Agreed, the name should be lowercase. I believe I wrote it like that so it would match the executable name, but the lowercase name is definitely more fitting with nixpkgs.